std/assembly/index.d.ts

```diff
@@ -34,7 +34,7 @@ declare type f32 = number;
 /** A 64-bit float. */
 declare type f64 = number;
 /** A 128-bit vector. */
-declare type v128 = object;
+declare type v128 = _V128;
```

Comment:
Seems this can use the name `V128` directly.

Reply:
Unfortunately, this doesn't work. Only `typeof V128` works normally

Reply:
I'm probably missing something, but what I was suggesting is basically
```ts
/** A 128-bit vector. */
declare type v128 = V128;

/** Pseudo-class representing the backing class of vector type. */
/** @internal */
declare class V128 {
  /** Converts this vector set to a string. */
  toString(radix?: number): string;
}
```
instead of having the indirection through `_V128`. To clarify perhaps, this doesn't work?

Reply:
Ah, right! Good suggestion. Fixed!